portability-transfer/src/main/java/org/dataportabilityproject/transfer/PortabilityInMemoryDataCopier.java

```diff
@@ -98,17 +96,15 @@ private void copyHelper(
     String jobIdPrefix = "Job " + jobId + ": ";
     logger.debug(jobIdPrefix + "copy iteration: {}", COPY_ITERATION_COUNTER.incrementAndGet());
 
-    // TODO: read in retry strategies from a config, but that's for later in v1
-    RetryStrategy expBackoffStrategy = new ExponentialBackoffRetryStrategy(5, 10, 2);
-    RetryStrategyLibrary library = new RetryStrategyLibrary(new LinkedList<>(), expBackoffStrategy);
+    RetryStrategyLibrary library = retryStrategyLibrary.get();
```

Comment:
nit: for a variable name, 'library' is pretty vague, how about just 'retryStrategy'?